net/FlatBuffers/Table.cs

```diff
@@ -83,15 +84,15 @@ protected int __vector(int offset)
         }
 
         // Initialize any Table-derived type to point to the union at the given offset.
-        protected TTable __union<TTable>(TTable t, int offset) where TTable : Table
+        public T __union<T>(int offset) where T : struct, Accessor
```

Comment:
Thanks, was not aware of this. This interface is mostly needed for unions, where we need to be able to initialize any accessor object type generically. Since structs don't support inheritance, I used an interface, which apparently has the same problem.
So how do I do anything to a template type which is a struct?
Alternatively, we could generate union accessors for each possible value of the union, but using templates would be cleaner.